### ISSUE TITLE
Conflicting IDE0001 and SA1135 for alias using statements

### ISSUE THREAD

**Issue by PBuchmann** [DotNetAnalyzers/StyleCopAnalyzers#2891]
Let A.B.C.D be a fully qualified type and shall be aliased like following:

```csharp
namespace A.B
{
    using T = A.B.C.D;
}
```

This Triggers IDE0001 (Name can be simplified). The simplified Version

```csharp
namespace A.B
{
    using T = C.D;
}
```

triggers SA1135 Using directive for type A.B.C.D should be qualified.

This is found in V1.1.1-rc.101

**Comment by sharwell** [comment]
IDE0001 is not produced by StyleCop Analyzers. SA1135 was introduced largely due to what many believed was reduced readability in code *caused* by IDE0001, so the discrepancy here is by design.
